vendor/licenses/git_submodule/witcherscript-grammar.dep.yml

```diff
@@ -0,0 +1,8 @@
+---
+name: witcherscript-grammar
+version: aa985201878f6d33eba88dd274e9480103d53294
+type: git_submodule
+homepage: https://github.com/ADawesomeguy/witcherscript-grammar.git
```

Comment:
Why not use the upstream repo https://github.com/nicollasricas/vscode-witcherscript directly?

Reply:
Grammar wasn't being detected properly so I just made a new repo based of the original grammar which I hope to be able to add to as well.

Reply:
For example I just (hopefully correctly) added RGB hex support, and I will (🤞🏽) continue to improve it.

Reply:
Think this can be resolved if that's fine?